src/Event/DynamoDb/DynamoDbRecord.php

```diff
@@ -0,0 +1,69 @@
+<?php declare(strict_types=1);
+
+namespace Bref\Event\DynamoDb;
+
+final class DynamoDbRecord
+{
+    /** @var array */
+    private $record;
+
+    /**
+     * @param mixed $record
+     */
+    public function __construct($record)
+    {
+        if (! is_array($record) || ! isset($record['eventSource']) || $record['eventSource'] !== 'aws:dynamodb') {
+            throw new \InvalidArgumentException('Event source must come from DynamoDB');
+        }
+
+        $this->record = $record;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getKeys()
+    {
+        return $this->record['dynamodb']['Keys'];
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getNewImage()
+    {
+        return $this->record['dynamodb']['NewImage'];
+    }
+
+    /**
+     * @return mixed|null
+     */
+    public function getOldImage()
+    {
+        return $this->record['dynamodb']['OldImage'] ?? null;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getSequenceNumber()
```

Comment:
I wonder if we can have more precise types in this class, for example here is it a string maybe?

Reply:
I will update that to be a string